packages/yoastseo/spec/scoring/assessments/seo/taxonomyTextLengthSpec.js

```diff
@@ -20,13 +20,14 @@ describe( "A TextLengthAssessment for a taxonomy page in Japanese", function() {
 		expect( assessment._config.veryFarBelowMinimum ).toEqual( assessmentConfigJapanese.taxonomyAssessor.veryFarBelowMinimum );
 	} );
 	it( "should return a good result for taxonomy pages in Japanese when the text is 60 characters or more", function() {
-		const paper = new Paper( "欧米では、かつては不吉の象徴とする迷信があり、魔女狩りなどによって黒猫が殺されることがあった。たとえばベルギー・ウェス。" );
+		const paper = new Paper( "欧米では、かつては不吉の象徴とする迷信があり、魔女狩りなどによって黒猫が殺されることがあった。その傾向は現在も続いており、" +
```

Comment:
Since punctuation was removed from the count, the sentence length became less than 60 characters, so I added some more text, in order to trigger the same feedback from the assessment.